.cirrus.yml

```diff
@@ -220,7 +220,7 @@ task:
     FILE_ENV: "./ci/test/00_setup_env_native_tsan.sh"
 
 task:
-  name: '[depends, sanitizers: memory (MSan)] [focal]'
```

Comment:
Did you mean to drop depends here?

Reply:
Yes. It is not possible to compile with MSan, but without depends, so this adds no information.

Reply:
Thanks, restored.